client/components/auth-form.jsx

```diff
@@ -25,18 +40,20 @@ const useStyles = makeStyles(theme => {
   };
 });
 
-export default function AuthForm() {
+export default function AuthForm(props) {
+  const { handleSignIn } = useContext(AppContext);
   const classes = useStyles();
+  const history = useHistory();
   const [formValues, setFormValues] = useState({
     username: '',
     password: ''
   });
-
   const [usernameError, setUsernameError] = useState(false);
   const [passwordError, setPasswordError] = useState(false);
-
   const [visibility, setVisibility] = useState(false);
 
+  const { route, setRoute } = props.props;
```

Comment:
You should be able to `useLocation` to get the `location.pathname`.

https://reactrouter.com/web/api/Hooks/uselocation